IronPython3Tests/IronPython3Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
IsTestProject is set automatically by consuming the microsoft test.sdk nuget
Not sure why you need to specify IsPackable false

Reply:
these were set by VS when creating a test project from template.